moveit_planners/chomp/chomp_motion_planner/src/chomp_planner.cpp

```diff
@@ -65,7 +65,13 @@ bool ChompPlanner::solve(const planning_scene::PlanningSceneConstPtr& planning_s
 
   if (not planning_scene->getRobotModel()->satisfiesPositionBounds(req.start_state.joint_state.position.data()))
   {
-    ROS_ERROR_STREAM_NAMED("chomp_planner", "Start state violates joint limits");
+    int i=0;
+    ROS_ERROR_STREAM_NAMED("chomp_planner", "Start state violates joint limits" << req.start_state.joint_state.position.data()[i++]);
+    ROS_ERROR_STREAM_NAMED("chomp_planner", "Start state violates joint limits" << req.start_state.joint_state.position.data()[i++]);
+    ROS_ERROR_STREAM_NAMED("chomp_planner", "Start state violates joint limits" << req.start_state.joint_state.position.data()[i++]);
+    ROS_ERROR_STREAM_NAMED("chomp_planner", "Start state violates joint limits" << req.start_state.joint_state.position.data()[i++]);
+    ROS_ERROR_STREAM_NAMED("chomp_planner", "Start state violates joint limits" << req.start_state.joint_state.position.data()[i++]);
+    ROS_ERROR_STREAM_NAMED("chomp_planner", "Start state violates joint limits" << req.start_state.joint_state.position.data()[i++]);
```

Comment:
i tihnk you need to rebase on master? or how did this get here? its unrelated.

Reply:
that was to test a problem in chomp, deleted them.